src/puppetbridge.ts

```diff
@@ -331,6 +331,12 @@ export class PuppetBridge extends EventEmitter {
 	public async start(callback?: () => Promise<void>) {
 		log.info("Starting application service....");
 
+		const AS = this.AS
+		process.on('SIGINT', function () {
+			AS.stop()
```

Comment:
please use an arrow function and then use this.AS instead. Also, check the linter, missing semicolumns and it requires to use doublequotes ^^

```
process.on("SIGINT", () => {
    this.AS.stop();
// etc.
});
```

Reply:
Updated.